plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -215,18 +218,28 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
       Logger.e(TAG, "Can't update source: style is not fully loaded.")
       return
     }
-    source?.let {
-      if (!style.styleSourceExists(it.sourceId)) {
+    source?.let { geoJsonSource ->
+      if (!style.styleSourceExists(geoJsonSource.sourceId)) {
         Logger.e(TAG, "Can't update source: source has not been added to style.")
         return
       }
+      annotations
+        .filter { it.value.getType() == AnnotationType.Symbol }
+        .forEach {
+          (it.value as Symbol).iconImageBitmap?.let {
+            val imagePlugin = image(Symbol.ICON_DEFAULT_NAME) {
```

Comment:
We are creating a style image with a fixed id. What will happen if we add multiple symbols using the bitmap api? They would share the same style image id and would be overriding eachother?

Reply:
Good catch